src/main/java/org/llorllale/cactoos/matchers/IsFalse.java

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
@dgroup [let's just implement `IsTrue`](https://github.com/llorllale/cactoos-matchers/issues/19#issuecomment-445479053)

Reply:
@llorllale Just for curiosity, why not implement an `IsFalse`? I think it's natural, because of symmetry. I know can I simulate an `IsFalse` doing `new IsNot<>(new IsTrue())` but isn't symmetric. Why not do an `IsFalse` class as a composition of  `new IsNot<>(new IsTrue())`?

Reply:
@fabriciofx it would be an arbitrary addition: "why not add `DoesNotHaveValues` opposite `HasValues`, or `DoesNotMatch` opposite `Matches`, etc?"

Reply:
@llorllale Right. I agree.